SalesforceCommerceCloud/pwa-kit
# Footer copyright: remove the remaining hardcoded year

This PR applies the same change from PR #741 to the remaining hardcoded year in the footer's copyright message. In particular, there were non `en-US` locales that needed the same treatment.


# How to Test-Drive This PR

1. `npm start` in the retail-react-app project
2. Scroll down to the footer section
3. Verify that the footer's copyright message looks all right: no duplicate year, and the year itself is correct (the current year)
4. Switch to other locales and also verify the year in the copyright message
